vertx-web-client/src/test/java/io/vertx/ext/web/client/WebClientTest.java

```diff
@@ -74,44 +85,72 @@ public void setUp() throws Exception {
     super.setUp();
     super.client = vertx.createHttpClient(new HttpClientOptions().setDefaultPort(8080).setDefaultHost("localhost"));
     client = WebClient.wrap(super.client);
+    clientWithInterceptors = (WebClientImpl) WebClient.wrap(super.client);
+    Map<HttpContext, String> uuids = new HashMap<>();
```

Comment:
I would remove the UUIDs thing as it is distracting and shows nothing except that headers work, which isn't something new in this change